io.openems.edge.evcs.webasto.unite/src/io/openems/edge/evcs/webasto/unite/api/Webasto.java

```diff
@@ -0,0 +1,228 @@
+		DATE(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		TIME(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		CHARGE_POINT_POWER(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT).accessMode(AccessMode.READ_ONLY)),
+		NUMBER_OF_PHASES(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		CHARGE_POINT_STATE(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		CHARGING_STATE(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		EQUIPMENT_STATE(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		CABLE_STATE(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		EVSE_FAULT_CODE(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		CURRENT_L1(Doc.of(OpenemsType.INTEGER).unit(Unit.MILLIAMPERE).accessMode(AccessMode.READ_ONLY)),
+		CURRENT_L2(Doc.of(OpenemsType.INTEGER).unit(Unit.MILLIAMPERE).accessMode(AccessMode.READ_ONLY)),
+		CURRENT_L3(Doc.of(OpenemsType.INTEGER).unit(Unit.MILLIAMPERE).accessMode(AccessMode.READ_ONLY)),
+		VOLTAGE_L1(Doc.of(OpenemsType.INTEGER).unit(Unit.VOLT).accessMode(AccessMode.READ_ONLY)),
+		VOLTAGE_L2(Doc.of(OpenemsType.INTEGER).unit(Unit.VOLT).accessMode(AccessMode.READ_ONLY)),
+		VOLTAGE_L3(Doc.of(OpenemsType.INTEGER).unit(Unit.VOLT).accessMode(AccessMode.READ_ONLY)),
+		ACTIVE_POWER_TOTAL(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT).accessMode(AccessMode.READ_ONLY)),
+		ACTIVE_POWER_L1(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT).accessMode(AccessMode.READ_ONLY)),
+		ACTIVE_POWER_L2(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT).accessMode(AccessMode.READ_ONLY)),
+		ACTIVE_POWER_L3(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT).accessMode(AccessMode.READ_ONLY)),
+		METER_READING(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		SESSION_MAX_CURRENT(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_ONLY)),
+		EVSE_MIN_CURRENT(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_ONLY)),
+		EVSE_MAX_CURRENT(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_ONLY)),
+		CABLE_MAX_CURRENT(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_ONLY)),
+		SESSION_ENERGY(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT_HOURS).accessMode(AccessMode.READ_ONLY)),
+		SESSION_START_TIME(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		SESSION_DURATION(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		SESSION_END_TIME(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_ONLY)),
+		FAILSAFE_CURRENT(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_WRITE)),
+		FAILSAFE_TIMEOUT(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_WRITE)),
+		CHARGING_CURRENT(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_WRITE)),
+		ALIVE_REGISTER(Doc.of(OpenemsType.INTEGER).accessMode(AccessMode.READ_WRITE));
```

Comment:
Optional: Best Practice would be, to break the method calls by adding //  behind every line

e.g.
CURRENT_L1(Doc.of(OpenemsType.INTEGER) //
                   .unit(Unit.MILLIAMPERE) //
                   .accessMode(AccessMode.READ_ONLY)), //